packages/web-runtime/src/composables/upload/useUpload.ts

```diff
@@ -104,7 +104,8 @@ export function useUpload(options: UploadOptions): UploadResult {
       ...(isTusSupported && {
         tusMaxChunkSize: unref(tusMaxChunkSize),
         tusHttpMethodOverride: unref(tusHttpMethodOverride),
-        tusExtension: unref(tusExtension)
+        tusExtension: unref(tusExtension),
+        xhrTimeout: store.getters.configuration?.options?.upload?.xhr?.timeout || 60000
```

Comment:
huh, why do you set the xhr timeout if tus is supported?

Reply:
Arrrghs 😅